lib/tasks/dev.rake

```diff
@@ -6,7 +6,24 @@ if Rails.env.development? || Rails.env.test?
     task prime: "db:setup" do
       include FactoryBot::Syntax::Methods
 
-      # create(:user, email: "user@example.com", password: "password")
+      create(:budget, user: User.create(
+        name: "admin",
+        email: "admin@example.com",
+        password: "secret",
+        admin: true
+      ))
```

Comment:
I would pull the admin use out into a variable, which then is used in creating this budget. i.e.:
```
admin = create(:user, :admin, name: "admin", email: "admin@example.com")
create(:budget, user: admin)
```

Reply:
A different question: does an admin have to have a budget?

Reply:
Good point, admin does not need a budget - simplifies things - done in sha: 7c2f098